mantidproject/mantid
# Extract Q-values from vertical axis of a MatrixWorkspace

Algorithm implemented as a python script, plus unit, system tests, and documentation

**To test:**
In MantidPlot, run the following code to extract the Q values for some of the files available for system tests:
```
workspace = Load("BASIS_59689_divided.dat")
print GetQsInQENSData(workspace)
```
Other files you can check:

> BASIS_59689_divided_sqw.nxs
> BASIS_56795_divided_sqw.nxs
> BASIS_56795_divided.dat
> irs26176_graphite002_red.nxs
> osiris97944_graphite002_red.nxs

Fixes #17835.

[Release notes](https://github.com/mantidproject/mantid/blob/4338b5e698ecfdb5919e3e8a30b8b6e8ec9122c2/docs/source/release/v3.9.0/indirect_inelastic.rst)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
